src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs

```diff
@@ -65,5 +66,105 @@ public void ExpandEnvironmentVariables()
                 Environment.SetEnvironmentVariable(env, null);
             }
         }
+
+        [Benchmark]
+        public void GetEnvironmentVariables()
+        {
+            PerfUtils utils = new PerfUtils();
+            string env = utils.CreateString(15);
+            try
+            {
+                // setup the environment variable so we can read it
+                Environment.SetEnvironmentVariable(env, "value");
+
```

Comment:
One thing that is worth following up on is where the warm up iteration gets discarded.  I think it happens in the BenchView script, so we probably want to move that to the harness so that local runs have the same behavior.